webpack.config.js

```diff
@@ -35,8 +35,12 @@ module.exports = {
     fs: 'empty'
   },
   externals: {
-    xhr2: 'XMLHttpRequest',
-    xmlhttprequest: 'XMLHttpRequest'
+    'xhr2': 'XMLHttpRequest',
+    'xmlhttprequest': 'XMLHttpRequest',
+    'node-fetch': 'fetch',
+    'text-encoding': 'TextEncoder',
+    'urlutils': 'URL',
+    'webcrypto': 'crypto'
```

Comment:
Are all these externals necessary? They're declared identically in [`solid-auth-oidc`'s webpack config](https://github.com/solid/solid-auth-oidc/blob/4cd999a1d4a7b0e763a2d90f179687f8df3a93e8/webpack.config.js).